### ISSUE TITLE
Intellisense broken for Conan packages

### ISSUE THREAD

**Issue by WoutProvost** [openmultiplayer/open.mp#609]
Intellisense for Conan packages is broken for me in Visual Studio.

So, I temporarily opened another editor (VSCode), to see if the same intellisense issue was present there, and it was. So, I tried to force that editor to use `~/.conan/data/**` as include path and the problem was resolved. Obviously this is not a good solution as it is IDE dependent.

This leads me to think there's something wrong with this piece in the `SDK/CMakeLists.txt` file:
```
# Conan third-party libraries
include(lib/cmake-conan/conan.cmake)
include(lib/cmake-conan/conan-omp.cmake)

conan_omp_add_lib(glm 0.9.9.8)
conan_omp_add_lib(robin-hood-hashing 3.11.5)
conan_omp_add_lib(span-lite 0.10.3)
conan_omp_add_lib(string-view-lite 1.6.0)
```

**Comment by Hual** [comment]
It's more likely an issue in conan's cmake generator itself, maybe what you said can be done from cmake though

**Comment by AmyrAhmady** [comment]
It is fixed in PR #671 , will be merged soon, you can give my branch a try @WoutProvost , reopn this issue if it still persists.
